src/main/kotlin/com/mparticle/kits/AppboyKit.kt

```diff
@@ -613,8 +613,18 @@ open class AppboyKit : KitIntegration(), AttributeListener, CommerceListener,
             }
         }
 
+        var sanitizedProductName: String = product.sku
```

Comment:
So looks like this logic is correct, but the setting from the server is poorly named (it's the opposite of what it says). Please add a comment above this line explaining that this behavior is intentional.